app/views/editor/wikiRich.html.erb

```diff
@@ -15,6 +15,25 @@
 
 <script src="/lib/publiclab-editor/dist/PublicLab.Editor.js"></script>
 
+<% if @is_new %>
+  <div class="container">
+    <div class="modal fade" id="wikiModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLongTitle" aria-hidden="true">
+      <div class="modal-dialog" role="document">
+        <div class="modal-content">
+        
+          <div class="modal-body"> 
+            <h1> Content from the feature will be added here! </h1>
```

Comment:
What if we skip the footer and just use a "Feature" as described in https://github.com/publiclab/plots2/blob/main/doc/DATA_MODEL.md#features -- that way site admins can just put in whatever text they need here. Actually couldn't we just put the entire container div in here, and just have a complete feature which just gets displayed on new wiki pages? That way it's totally customizable! From line 19-34 would be replaced by the "feature" code as described in https://github.com/publiclab/plots2/blob/main/doc/DATA_MODEL.md#features

Reply:
Sure @jywarren! Even I was thinking of using a `feature` there but I'm not really sure where I need to add the HTML code for the feature. Could you please help me out with that?